CHANGELOG.md

```diff
@@ -1,12 +1,16 @@
 For 1.x changes, go [here](https://github.com/twilio/twilio-video.js/blob/support-1.x/CHANGELOG.md).
 
-2.0.0-beta16 (in progress)
-==========================
+2.0.0-beta16 (December 10, 2019)
+================================
 
 New Features
-------------
-- You can now specify the mode to control track switch off behavior by specifying a
-  property `trackSwitchOffMode` in BandwidthProfile options. This can be set to one of
+ ------------
+
+- This release supports all the features of the [Track Priority and Bandwidth Profile APIs](https://www.twilio.com/docs/video/migrating-1x-2x#track-priority-and-bandwidth-profiles-group-rooms-only-private-beta).
+
+- You can now specify the mode to control Track switch off behavior by specifying a
+  property `trackSwitchOffMode` in BandwidthProfileOptions (JSDK-2549).
+  `trackSwitchOffMode` can be set to one of
```

Comment:
@makarandp0 Can you make sure the ticket is at the end the sentence for `disabled`?